From aebe5f9bf0a714b190fcd51a57e2b2ca3abf9bac Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Wed, 17 Oct 2018 18:34:44 -0700
Subject: [PATCH] TRITON-880 want ufds to store and understand Yubikey
 attestation certs Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
 Approved by: Cody Peter Mello <cody.mello@joyent.com>

---
 scripts/sdc-init.js | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/scripts/sdc-init.js b/scripts/sdc-init.js
index 23b4836a..56cb887c 100755
--- a/scripts/sdc-init.js
+++ b/scripts/sdc-init.js
@@ -116,6 +116,9 @@ function translateConfig(cb) {
     sdcExtras.metadata['sapi-url'] = 'http://' + self.config.sapi_domain;
     sdcExtras.metadata['assets-ip'] = self.config.assets_admin_ip;
 
+    // default-on feature flags
+    sdcExtras.metadata['UFDS_ATTESTATION'] = true;
+
     return cb(null);
 
     // XXX - other things that aren't metadata?
-- 
2.21.0

